api/acknowledgements.txt

```diff
@@ -2,7 +2,8 @@
 // Creative Commons Attribution 4.0 International License; see
 // http://creativecommons.org/licenses/by/4.0/
 
-*Acknowledgements*
+[float]
```

Comment:
I have tried building both with and without this `[float]` attribute.  The only visible effect that I saw is inclusion in the table of contents or not, why remove it?  The effect is most visible (to me) for the HTML output as I'm more likely to use the table of contents to browse there.

Reply:
I was concerned that without the `[float]` attribute the chapter numbering might be messed up, but everything looks good without it, so I've removed it.  The "Acknowledgements" now shows up in the table of contents, without a chapter number, which seems reasonable.  Thanks!

Reply:
Okay, thanks.